source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -1,12 +1,141 @@
 =================
-Insert a Document
+Insert Operations
 =================
 
 .. default-domain:: mongodb
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
 Overview
 --------
 
-In this section, we show you how to call the write operations to **insert**
-documents from a collection in your MongoDB database.
+In this guide, you will learn how to insert documents into MongoDB. 
+
+You can use MongoDB to retrieve, update and delete information. To
+perform any of those operations, that information, such as user profiles
+and orders, needs to exist in MongoDB. For that information to exist,
+you need to first perform an insert operation.
+
+An insert operation inserts a single or multiple documents in MongoDB
+using the  ``insertOne()``, ``insertMany()`` and ``bulkWrite()``
+methods. 
+
+The following sections focus on ``insertOne()`` and
+``insertMany()``. For information on how to use the ``bulkWrite()``
+method, see our ``Bulk Operations <TODO>`` page. 
+
+Insert a Single Document
+------------------------
+
+When you want to insert a single document, use the ``insertOne()``
+method. If the insertion is successful,  the method returns an
```

Comment:
I think this can be reworded to use imperative mood and make it feel a touch more authoritative. Consider if someone search "insert a document into mongodb", which would immediately answer their question?
> When you want to insert a single document, use the ``insertOne()`` method.
vs
> Use the ``insertOne()`` method when you want to insert a single document.

```suggestion
Use the ``insertOne`` method when you want to insert a single document. If the insertion is successful,  the method returns an
```

Reply:
I wonder if we can add more flavor about the method here rather than immediately jumping into what it returns. The information in the note about automagic `_id` may fit well here? Is there any other behavior we should state? Can it error? How?